test/algorithms/buffer/test_buffer_geo.hpp

```diff
@@ -65,25 +62,53 @@ void test_one_geo(std::string const& caseid,
             strategy,
             expected_count, expected_holes_count, expected_area,
             settings);
+
+    if (symmetric && distance_left > 0.0)
+    {
+        // Verify if all the points of the output geometry are at or around the buffered distance
+        // For linestrings with flat ends, it's not necessarily the case, there may be points
+        // too close, especially on artefacts in heavily curved input with flat ends.
+        // Therefore the default expectation can be modified. Inspect the SVG visually before doing this.
+        std::size_t too_close = 0, too_far = 0, total = 0;
```

Comment:
we use one declaration per line style, right?

Reply:
Indeed, thanks (and I applied also your other indentation comments)